org.freedesktop.fwupd.json

```diff
@@ -0,0 +1,211 @@
+          "sources": [
+            {
+                "type": "archive",
+                "url": "http://kernel.org/pub/linux/utils/kernel/hotplug/udev-175.tar.bz2",
+                "sha256": "4c7937fe5a1521316ea571188745b9a00a9fdf314228cffc53a7ba9e5968b7ab"
+            },
+            {
+                "type": "script",
+                "dest-filename": "autogen.sh",
+                "commands": [
+                    "autoreconf -vfi"
+                ]
+            }
+          ]
+        },
+        {
+            "name": "libusb",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/libusb/libusb/releases/download/v1.0.22/libusb-1.0.22.tar.bz2",
+                    "sha256": "75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157"
+                }
+            ]
+        },
+        {
+            "name": "gusb",
+            "buildsystem": "meson",
+            "config-opts": ["-Ddocs=false",
+                            "-Dvapi=false",
+                            "-Dtests=false"],
+            "cleanup": [
+                "/bin/gusbcmd"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://people.freedesktop.org/~hughsient/releases/libgusb-0.3.0.tar.xz",
+                    "sha256": "d8e7950f99b6ae4c3e9b8c65f3692b9635289e6cff8de40c4af41b2e9b348edc"
+                }
+            ]
+        },
+        {
+            "name": "efivar",
+            "buildsystem": "simple",
+            "build-commands": ["make prefix=/app libdir=/app/lib", "make install prefix=/app libdir=/app/lib"],
+            "cleanup": [
+                "/bin/efivar"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/rhboot/efivar/releases/download/35/efivar-35.tar.bz2",
+                    "sha256": "1e033dc5d099a44fd473b0887dbcc4b105613efab0fb3c5df9f111ea5d147394"
+                }
+            ]
+        },
+        {
+            "name": "libsmbios_c",
+            "config-opts": ["--disable-doxygen",
+                            "--disable-graphviz",
+                            "--disable-python"],
+            "cleanup": [
+                "/sbin/smbios*",
+                "/share/locale/*/LC_MESSAGES/libsmbios.mo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/dell/libsmbios/archive/v2.4.2.tar.gz",
+                    "sha256": "ebfe18415e24bbec06d0a9ea1066c8dcd82982555373712713d7e194138650de"
+                }
+            ]
+        },
+        {
+            "name": "gnu-efi",
+            "buildsystem": "simple",
+            "build-commands": ["make", "make PREFIX=/app install"],
```

Comment:
```json
"no-autogen": true,
"make-install-args": ["PREFIX=/app"]
```

Reply:
Fixed, thanks.

Reply:
That is fine, `make-args` and `make-install-args` are for that.

Reply:
No, `make-install-args` sets up `PREFIX=/app` in the builder environment. This would be like specifying `PREFIX=/app make install` but that doesn't work, as gnu-efi expects a PREFIX argument in `argv` rather than reading the environment in `envp`.